crt0stack/src/builder.rs

```diff
@@ -51,8 +51,14 @@ impl<'a> Handle<'a> {
     /// This reference can be used as the initial stack pointer
     /// to execute a Linux ELF binary.
     #[inline]
-    pub fn start_ptr(&self) -> &'a () {
-        unsafe { &*(&self.0[self.1] as *const u8 as *const ()) }
+    pub fn start_ptr(&self) -> &'a Stack {
+        #[repr(C, align(16))]
+        struct Aligned(u128);
```

Comment:
Why not take the already defined `Stack` which is already defined? Is it because it is unsized?

Reply:
Yes